docs/code-samples/getting-started/daml/Scripts/Holding.daml

```diff
@@ -141,3 +145,24 @@ createParty name = do
   userId <- validateUserId name
   createUser (User userId (Some party)) [CanActAs party]
   pure party
+
+-- | Create a holding factory with reference.
```

Comment:
```suggestion
-- | Create a holding factory as well as the companion `Reference` template.
```

I would also add a link to the page in the docs where the reference pattern is explained